src/components/pages/Offers/Offer/Stocks/StockItemOptionsMenu/StockItemOptionsMenu.jsx

```diff
@@ -0,0 +1,84 @@
+import { Menu, MenuButton, MenuItem, MenuList } from '@reach/menu-button'
+import PropTypes from 'prop-types'
+import React, { useCallback } from 'react'
+
+import { ReactComponent as OptionMenuIcon } from 'icons/ico-more-horiz.svg'
+
+import { ReactComponent as AddActivationCodeIcon } from './assets/add-activation-code-light.svg'
+import { ReactComponent as DeleteStockLightIcon } from './assets/delete-stock-light.svg'
+
+const StockItemOptionsMenu = ({
+  canAddActivationCodes,
+  deleteButtonTitle,
+  deleteStock,
+  disableDeleteButton,
+  isNewStock,
+  isOfferDisabled,
+}) => {
+  const addActivationCodes = useCallback(() => {}, [])
+
+  return (
+    <Menu className="stock-options-menu-container">
+      <MenuButton
+        className="stock-options-menu-button"
+        disabled={isOfferDisabled}
+        title="Opérations sur le stock"
+        type="button"
+      >
+        <OptionMenuIcon
+          alt="Opérations sur le stock"
+          className="stock-options-menu-button-icon"
+        />
+      </MenuButton>
+      <MenuList className="stock-options-menu-list">
+        <MenuItem
+          className="stock-options-menu-item"
+          disabled={disableDeleteButton}
+          onSelect={deleteStock}
+          title={deleteButtonTitle}
+        >
+          <DeleteStockLightIcon
+            alt="Supprimer le stock"
+            aria-hidden
+            className="stock-options-menu-item-icon"
+          />
+          <span className="stock-options-menu-item-text">
+            {'Supprimer le stock'}
+          </span>
+        </MenuItem>
+        {canAddActivationCodes && (
+          <MenuItem
+            className="stock-options-menu-item"
+            disabled={!isNewStock}
+            onSelect={addActivationCodes}
+            title={
+              isNewStock
+                ? "Ajouter des codes d'activation"
+                : "Les stocks déjà existants ne peuvent pas recevoir de codes d'activations"
```

Comment:
```suggestion
                : "Les stocks déjà existants ne peuvent pas recevoir de codes d'activation"
```

Je crois que le deuxième mot ne prend pas de s, mais je ne suis pas 100% confiant :monocle_face:

Reply:
Sauf s'il y a plusieurs activations, mais ça n'est pas le use case à priori :sweat_smile: